hs/src/Reach/AST/DLBase.hs

```diff
@@ -333,9 +339,14 @@ instance PrettySubst DLArg where
 
 staticZero :: DLArg -> Bool
 staticZero = \case
-  DLA_Literal (DLL_Int _ 0) -> True
+  DLA_Literal (DLL_Int _ _ 0) -> True
   _ -> False
 
+uintTyMax :: UIntTy -> DLArg
+uintTyMax = \case
+  True -> DLA_Literal $ DLL_Int sb True $ uint256_Max
```

Comment:
Why not a constant?

Reply:
"Constants" in Reach actually variable across the whole system but constant on a signle platform